MEETINGS.md

```diff
@@ -1,32 +1,27 @@
-# GitOps Working Group Meetings
-The GitOps Working group typically meets on the second Thursday of each month at 18:00 GMT - this meeting can be found in the [CNCF calendar](https://www.cncf.io/calendar/).
-[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+# Meetings
 
-## Upcoming Meetings
-### Thursday, April 15th at 18:00 GMT
-- [Video Conference Link](https://zoom.us/my/cncfgitopswg?pwd=aE9FMjR0M1djd3huMlNUMWZXSXpyQT09)
-- Meeting Passcode: `77777`
+The GitOps Working Group typically meets on the second Thursday of each month at 18:00 GMT, and are listed below. This meeting is also on the [CNCF calendar](https://www.cncf.io/calendar/).
 
-## Past Meetings
-### Thursday, March 11th at 18:00 GMT
+Additionally Working Group Committees meet as needed to complete their projects, and are also listed below.
 
-- [Video Recording](https://zoom.us/rec/share/SYMUyXzmVF4eLmO9d2N3AOQ3ror5cJo361pnz_PtxUnDMio0vXRd-qpjqMHPnGeo.J_6sc5Mv8IIKi9jr?startTime=1615486031000)
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+See the [Google Doc](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) for detailed join info, agenda, and notes for all GitOps Working Group meetings.
 
-### Thursday, February 11th at 18:00 GMT
+## Working Group Monthly
 
-- [Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)
-- Passcode: `fwPy8^fb`
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+- May 13, 2021: [join link and agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.lq3s56hyydm0) ⚠️ Note: May Monthly meeting moved to 3rd Thursday due to KubeCon EU
```

Comment:
```Suggestion
- May 13, 2021: [join link and agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.lq3s56hyydm0)
```
@scottrigby May 13th is the second Thursday, whereas KubeCon EU is during the first Thursday according to https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/

Reply:
You are correct 😅